features/aanhef.feature

```diff
@@ -97,67 +106,80 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente briefaanhef in com
+      | Eigen                 | man                 | GA VV GN            | F. Groenen                | Geachte heer Groenen                   |
+      | Eigen                 | onbekend            | GA VL VV GN         | C.F.H. van Velzen         | Geachte C.F.H. van Velzen              |
+      | Partner na eigen      | vrouw               | GA VV GN-VP GP      | I. van Velzen-in het Veld | Geachte mevrouw Van Velzen-in het Veld |
+      | Partner na eigen      | vrouw               | GA VV GN-VP GP      | F. Groenen-Groenink       | Geachte mevrouw Groenen-Groenink       |
+      | Partner               | vrouw               | GA VP GP            | S. van Velzen             | Geachte mevrouw Van Velzen             |
+      | Partner               | vrouw               | GA VP GP            | J.F.R. Groenen            | Geachte mevrouw Groenen                |
+      | Partner voor eigen    | man                 | GA VP GP-VV GN      | F. in het Veld-van Velzen | Geachte heer In het Veld-van Velzen    |
+      | Partner voor eigen    | vrouw               | GA VP GP-VV GN      | M. Groenen-Groenink       | Geachte mevrouw Groenen-Groenink       |
+      | Partner voor eigen    | onbekend            | GA VL. VP GP-VV GN  | J.P. van Velzen-Groenen   | Geachte J.P. van Velzen-Groenen        |
 
     Voorbeelden: voorvoegsels met hoofdletter of kleine letter
       | aanduidingAanschrijving | geslachtsaanduiding | VV     | GN     | VP     | GP     | aanhef                                 |
-      | E                       | man                 | In het | Veld   | van    | Velzen | Geachte heer In het Veld               |
-      | N                       | vrouw               | van    | Velzen | In het | Veld   | Geachte mevrouw Van Velzen-In het Veld |
-      | P                       | vrouw               | In het | Veld   | van    | Velzen | Geachte mevrouw Van Velzen             |
-      | V                       | man                 | van    | Velzen | In het | Veld   | Geachte heer In het Veld-van Velzen    |
+      | Eigen                   | man                 | In het | Veld   | van    | Velzen | Geachte heer In het Veld               |
+      | Partner na eigen        | vrouw               | van    | Velzen | In het | Veld   | Geachte mevrouw Van Velzen-In het Veld |
+      | Partner                 | vrouw               | In het | Veld   | van    | Velzen | Geachte mevrouw Van Velzen             |
+      | Partner voor eigen      | man                 | van    | Velzen | In het | Veld   | Geachte heer In het Veld-van Velzen    |
+      | Partner voor eigen      | onbekend            | van    | Velzen | In het | Veld   | Geachte M.L. van Velzen-In het Veld    |
 
     Voorbeelden: adelijke titels
-      | adellijkeTitel_predikaat | aanduidingNaamgebruik | aanschrijfwijze                             | aanhef                    |
```

Comment:
Bij aanschrijfwijze komt er een attribuut bij: RegelVoorafgaandAanAanschrijfwijze met als description: deze regel moet als aparte regel boven de aanschrijfwijze worden geplaatst. Komt alleen voor bij personen met een adellijke titel of predicaat.  

website hoge raad van adel:
Wanneer de geadresseerde recht heeft op een predicaat, dan krijgt dit als eerste woord van de adressering een hoofdletter. Het gebruik van de woorden ‘De heer’ en ‘Mevrouw’ naast een titel of predikaat is dubbelop en daarom onjuist.

De adressering aan graven en burggraven luidt 'De hooggeboren heer', die  aan gravinnen en burggravinnen 'De hooggeboren vrouwe'. Een brief begint dan met 'Hooggeboren heer / vrouwe', gevolgd door een komma, een witregel en de briefinhoud:

De hooggeboren heer
A.B.C. graaf van Limburg Stirum
Voor de overige edellieden is de adressering 'De hoogwelgeboren heer / vrouwe' en de briefaanhef 'Hoogwelgeboren heer / vrouwe', gevolgd door een komma, een witregel en de briefinhoud:

De hoogwelgeboren vrouwe
A.B.C. barones van Dedem
De hoogwelgeboren heer
Jonkheer A.B.C. Verheyen.